chapter_linear-networks/image-classification-dataset.md

```diff
@@ -280,12 +280,21 @@ def load_data_fashion_mnist(batch_size, resize=None):  #@save
 #@tab tensorflow
 def load_data_fashion_mnist(batch_size, resize=None):  #@save
     """Download the Fashion-MNIST dataset and then load into memory."""
-    # TODO: Resize
     (mnist_train_x, mnist_train_y), (mnist_test_x, mnist_test_y) = tf.keras.datasets.fashion_mnist.load_data()
-    return (
+    if(resize is None):
+      return (
         tf.data.Dataset.from_tensor_slices(
             (mnist_train_x, mnist_train_y)).batch(batch_size).shuffle(len(mnist_train_x)),
         tf.data.Dataset.from_tensor_slices((mnist_test_x, mnist_test_y)).batch(batch_size))
+    else:
+      def map_fn(img,label):
+        img = tf.reshape(img, (28,28,1))
+        img =  tf.image.resize(img, (resize))
+        return (img,label)
+      return (
+        tf.data.Dataset.from_tensor_slices(
```

Comment:
Move the common code to above.

Reply:
I'm not getting what is the common code, here.